mlrun/api/rundb/sqldb.py

```diff
@@ -222,9 +228,10 @@ def list_artifacts(
             best_iteration=best_iteration,
             kind=kind,
             category=category,
+            tree=tree,
```

Comment:
```suggestion
            producer_id=tree,
```

Reply:
As discussed, I don't want to change the RunDB interface